src/Packet.cpp

```diff
@@ -151,6 +151,44 @@ void Packet::copyInto(Packet& pack) {
   strcpy(pack.data, data);
 }
 
+void Packet::setPacketData(char charArray[240]){
+  strcpy(data, charArray);
+}
+
+void Packet::setPacketData(String wordy){
+  char charArray[240];
+  wordy.toCharArray(charArray, sizeof(wordy));
+  this->setPacketData(charArray);
```

Comment:
Unneccessary `this` pointer (it is implied when calling member method `setPacketData` from inside its own class).
The same goes for all `this` pointers in this file.